<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 154, 191);
background-color: rgb(253, 245, 223);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="mainColumn">
     <item alignment="Qt::AlignVCenter">
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>950</width>
         <height>650</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2000</width>
         <height>1600</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="column" stretch="1,3,1">
          <property name="leftMargin">
           <number>40</number>
          </property>
          <property name="rightMargin">
           <number>40</number>
          </property>
          <property name="bottomMargin">
           <number>40</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,4">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Welcome</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="outNameSurname">
              <property name="styleSheet">
               <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Name Surname</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>80</number>
            </property>
            <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
             <widget class="QGroupBox" name="groupBox">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>220</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>220</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 170, 127);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
border-radius: 6;
border: 2px solid white;
color: rgb(255, 255, 255);

</string>
              </property>
              <property name="title">
               <string>Your information</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <widget class="QWidget" name="horizontalLayoutWidget_2">
               <property name="geometry">
                <rect>
                 <x>40</x>
                 <y>20</y>
                 <width>311</width>
                 <height>80</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item alignment="Qt::AlignLeft">
                 <widget class="QLabel" name="label_3">
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Phone number:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="outPhoneNumber">
                  <property name="maximumSize">
                   <size>
                    <width>150</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>05426147696</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="horizontalLayoutWidget_3">
               <property name="geometry">
                <rect>
                 <x>40</x>
                 <y>110</y>
                 <width>311</width>
                 <height>81</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item alignment="Qt::AlignLeft">
                 <widget class="QLabel" name="label_5">
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Email Address:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="outEmail">
                  <property name="maximumSize">
                   <size>
                    <width>150</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>testuser@gmail.com</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item alignment="Qt::AlignRight|Qt::AlignVCenter">
             <widget class="QWidget" name="widget_2" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>190</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>200</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 2px solid white;
border-radius: 8;
background-color: rgb(0, 170, 127);</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item alignment="Qt::AlignHCenter">
                  <widget class="QCommandLinkButton" name="btnLogout">
                   <property name="minimumSize">
                    <size>
                     <width>150</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 170, 127);</string>
                   </property>
                   <property name="text">
                    <string>LOGOUT</string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignHCenter">
                  <widget class="QPushButton" name="btnProfile">
                   <property name="minimumSize">
                    <size>
                     <width>150</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="text">
                    <string>Profile</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>40</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnDocuments">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>80</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
border-radius: 8;</string>
              </property>
              <property name="text">
               <string>Documents</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAnnouncements">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>80</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
border-radius: 8;</string>
              </property>
              <property name="text">
               <string>Announcements</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnExperiences">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>80</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
border-radius: 8;</string>
              </property>
              <property name="text">
               <string>Experiences</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
